peachjam/models/ingestors.py

```diff
@@ -74,3 +91,11 @@ def get_adapter(self):
         ingestor_settings = IngestorSetting.objects.filter(ingestor=self)
         settings = {s.name: s.value for s in ingestor_settings}
         return klass(self, settings)
+
+    def queue_task(self):
+        run_ingestor(self.id, repeat=self.repeat, schedule=self.schedule)
+
+    def save(self, *args, **kwargs):
+        super().save(*args, **kwargs)
+        if self.enabled:
+            self.queue_task()

```

Comment:
@longhotsummer we have an infinite loop here. we call queue_task on save, but when we run the task, we update the last_refreshed, which calls save, entering a loop. We can schedule the task to run five minutes out, but the loop is there

Reply:
that works. done